packages/core/src/views/shared/WalletAppBadge.svelte

```diff
@@ -78,27 +78,27 @@
   }
 
   .background-gray {
-    background: var(--onboard-gray-500, var(--gray-500));
+    background: var(--onboard-wallet-app-icon-background, var(--onboard-gray-500, var(--gray-500)));
```

Comment:
```suggestion
    background: var(--onboard-wallet-app-icon-background-gray, var(--onboard-gray-500, var(--gray-500)));
```
We will want to make each more independent as far as the variable name is concerned. I think post-fixing the variable with the class color type can achieve this